Content.Server.Database/ModelSqlite.cs

```diff
@@ -144,8 +148,11 @@ private static byte[] JsonDocumentToByteArray(JsonDocument? document)
             return stream.ToArray();
         }
 
-        private static JsonDocument ByteArrayToJsonDocument(byte[] str)
+        private static JsonDocument? ByteArrayToJsonDocument(byte[]? str)
         {
+            if (str == null || str.Length == 0)
+                return null;
```

Comment:
instead of making all nullable maybe better to return empty JsonDocument?

`JsonDocument.Parse("{}");` or `JsonDocument.Parse(string.Empty)`

Also needs commentaries

```suggestion
            // Cosmatic-Draft Record System-start
            if (str == null || str.Length == 0)
                return null;
            // Cosmatic-Draft Record System-end
```